pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -68,28 +72,34 @@ buildPythonPackage rec {
     cached-property
     clickhouse-driver
     dask
+    datafusion
+    duckdb
+    duckdb-engine
     graphviz
+    importlib-metadata
     multipledispatch
     numpy
     pandas
     parsy
+    poetry-dynamic-versioning
     pyarrow
+    pydantic
     pytz
     regex
     requests
     sqlalchemy
-    tables
+    tabulate
     toolz
-  ] ++ lib.optionals (pythonOlder "3.8" && lib.versionOlder version "3.0.0") [
-    importlib-metadata
   ];
 
   checkInputs = [
     pytestCheckHook
     click
     pytest
+    pytest-benchmark
     pytest-mock
     pytest-xdist
+    sqlite
   ];
 
   postPatch = ''
```

Comment:
postPatch can be removed.
substituteInPlace doesn't replace either of those dependencies